mods/cnc/rules/infantry.yaml

```diff
@@ -15,6 +15,7 @@ E1:
 	Armament:
 		Weapon: M16
 	AttackFrontal:
+		Voice: Move
```

Comment:
> It may be cleaner to define an Attack voice instead of overriding this everywhere.

That will add redundancy.

> Otherwise, `Move` should really be renamed to something more generic like `Action`.

I decided against it for compatibility reasons and pull request scope.